src/Servers/IIS/IIS/src/Core/ThrowingWasUpgradedWriteOnlyStream.cs

```diff
@@ -8,20 +8,31 @@
 
 namespace Microsoft.AspNetCore.Server.IIS.Core
 {
+    /// <summary>
+    /// A <see cref="Stream"/> which throws on calls to write after the stream was upgraded
+    /// </summary>
+    /// <remarks>
+    /// Users should not need to instantiate this class.
+    /// </remarks>
     public class ThrowingWasUpgradedWriteOnlyStream : WriteOnlyStream
```

Comment:
Why is this public?

Reply:
It shouldn't be, but I probably can't make it private this late in the game.